components/pedigree/resources/src/main/resources/pedigree/view/exportSelector.js

```diff
@@ -148,7 +193,11 @@ define([
 
             console.log("Export data: >>" + exportString + "<<");
             if (exportString != "") {
-                FileSaver.saveTextAs(exportString, fileName);
+                if (exportType != "image") {
+                    FileSaver.saveTextAs(exportString, fileName);
+                } else if (exportType == "image" && imageType == "svg") {
+                    FileSaver.saveTextAs(exportString, fileName, 'base64', 'data:image/png;base64');
```

Comment:
my bad for misreading function name, but the general argument stands that IMHO it makes ore sense to have two functions which only take user-relevant parameters (data and filename) while what we export should be somehow encoded in function name. The original idea was not "save text as <TYPE>" but "save text as <FILENAME>". I guess bad naming to start with